packages/fuel-gauge/src/funding-transaction.test.ts

```diff
@@ -51,14 +46,26 @@ describe(__filename, () => {
   };
 
   it('should successfully fund a transaction request when it is not fully funded', async () => {
-    const sender = Wallet.generate({ provider });
-    const receiver = Wallet.generate({ provider });
+    const initialAmount = 500_000;
+    using launched = await launchTestNode({
+      walletsConfig: {
+        count: 2,
+        amountPerCoin: initialAmount,
+      },
+    });
```

Comment:
```suggestion
    using launched = await launchTestNode({
      walletsConfig: {
        amountPerCoin: initialAmount,
      },
    });
```
The default is two wallets.